htmd/parameterization/cli.py

```diff
@@ -67,18 +67,24 @@ def main_parameterize(arguments=None):
     import numpy as np
     import math
 
-    def printEnergies(m):
-        print("\n == Diagnostic Energies == ")
+    def printEnergies(m, filename):
+        fener = open(filename, "w")
```

Comment:
The file isn't closed!

Reply:
Fixed in https://github.com/Acellera/htmd/pull/423/commits/76e524779a1b9e79ca770ab0289b59a4991e2488